graphql-server/go-server/minio/client.go

```diff
@@ -0,0 +1,77 @@
+package minio
+
+import (
+	"sync"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+)
+
+var minioClient *minio.Client = nil
+var coreClient *minio.Core = nil
+
+var mutex *sync.Mutex
+
+var (
+	minioAddress         = "kubeagi-minio.172.22.96.136.nip.io"
+	minioAccessKeyId     = "admin"
+	minioSecretAccessKey = "Passw0rd!"
+	minioSecure          = false
+	minioBucket          = "arcadia"
+	minioBasePath        = "datasource"
```

Comment:
ENV ?

Reply:
Currently arcadia deployed via helm will have a configmap named `<release-name>-config`, which records information about the configuration Datasource for recording minio.

Here it is recommended to configure the flag parameter to support user configuration, if it is not set, just read it from the corresponding `configmap`.

Reply:
see pkg/config to find some helper functions

Reply:
@dayuy see @Abirdcfly